go/types/perf/perf_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package perf
```

Comment:
Note this needs to be in a separate package because the perf suite relies on the types package, so types can't have a perf suite.

The same will be true for datas, dataset, etc as well. We'll need to use this idiom a bunch.